airflow/bin/cli.py

```diff
@@ -1720,6 +1720,9 @@ class CLIFactory:
             "Do not prompt to confirm reset. Use with care!",
             "store_true",
             default=False),
+        'default_connection': Arg(
+            ("-d", "--default_connection"), "Flag to create airflow default connections",
+            default=True),
```

Comment:
If this is true by default can it be turned off?

Reply:
if we call cli like airflow initdb --default_connection False

Reply:
More normal would be `--no-default-connections` (a separate flag to turn it off/on)

Reply:
no need i guess, --default_connection is enough why we need two variables here

Reply:
what me to change the variable name from --default-connection to --no-default-connections?

Reply:
I agree with @ashb. It seems more intuitive to me.